src/main/java/com/capitalone/dashboard/service/TestResultServiceImpl.java

```diff
@@ -330,7 +330,7 @@ public String createTest(TestCreateRequest request) throws HygieiaException {
 
             testResult = createTestJunit(request);
         } else {
-            return "we are not accepting " + request.getTestType() + " sourceFormat " + request.getSourceFormat();
+            return "Hygieia are not accepting " + request.getTestType() + " sourceFormat " + request.getSourceFormat();
```

Comment:
can we change the verbiage to `Hygieia does not support` instead of `Hygieia are not accepting`